docs/tasks/debug-application-cluster/audit.md

```diff
@@ -70,3 +147,105 @@ the file name (before the file extension) and create a new audit log file.
 Kubernetes may delete old log files when creating a new log file; you can configure
 how many files are retained and how old they can be by specifying the `audit-log-maxbackup`
 and `audit-log-maxage` options.
+
+Logged events are printed as:
+
+```
+2017-06-15T21:50:50.259470834Z AUDIT: id="591e9fde-6a98-46f6-b7bc-ec8ef575696d" stage="RequestReceived" ip="10.2.1.3" method="update" user="system:serviceaccount:kube-system:default" groups="\"system:serviceaccounts\",\"system:serviceaccounts:kube-system\",\"system:authenticated\"" as="<self>" asgroups="<lookup>" namespace="kube-system" uri="/api/v1/namespaces/kube-system/endpoints/kube-controller-manager" response="<deferred>"
+2017-06-15T21:50:50.259470834Z AUDIT: id="591e9fde-6a98-46f6-b7bc-ec8ef575696d" stage="ResponseComplete" ip="10.2.1.3" method="update" user="system:serviceaccount:kube-system:default" groups="\"system:serviceaccounts\",\"system:serviceaccounts:kube-system\",\"system:authenticated\"" as="<self>" asgroups="<lookup>" namespace="kube-system" uri="/api/v1/namespaces/kube-system/endpoints/kube-controller-manager" response="200"
+```
+
+If the `AdvancedAuditing` feature flag is not enabled, the audit log reverts to its legacy format:
+
+```
+2017-03-21T03:57:09.106841886-04:00 AUDIT: id="c939d2a7-1c37-4ef1-b2f7-4ba9b1e43b53" ip="127.0.0.1" method="GET" user="admin" groups="\"system:masters\",\"system:authenticated\"" as="<self>" asgroups="<lookup>" namespace="default" uri="/api/v1/namespaces/default/pods"
+2017-03-21T03:57:09.108403639-04:00 AUDIT: id="c939d2a7-1c37-4ef1-b2f7-4ba9b1e43b53" response="200"
+```
+
+### Audit webhook
+
+The audit webhook backend can be used to have [kube-apiserver][kube-apiserver]
+send audit events to a remote service. The webhook requires the `AdvancedAuditing`
+feature flag and is configured using the following command line flags:
+
+```
+--audit-webhook-config-file=/etc/kubernetes/audit-webhook-kubeconfig
+--audit-webhook-mode=batch
+```
+
+`audit-webhook-mode` controls caching strategies used by the webhook. Known modes are:
```

Comment:
nit: s/caching/queuing/ ? I'm not sure queuing is right, but I don't think caching is...